docs/source/dev_guide/algorithms/logistic_regression/logistic-regression.rst

```diff
@@ -166,17 +166,20 @@ parameters:
    * - penaltyL1
      - :math:`0`
      - L1 regularization coefficient
+
+       .. note:: This parameter is not supported on GPU.
```

Comment:
Very unclear. It should affect linkage with selected solvers.

Reply:
That was the input

Reply:
And there is. We say that we support only one solver - SGD-minibatch for GPU. And we do not in any way support the L1 regulation of GPU.

Reply:
I can change this note to "L1 regularization is not supported on GPU." if it makes it clearer